tests/sessions_tests/test_checks.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+
+from django.conf import settings
+from django.contrib.sessions.checks import check_names_conflict
+from django.core import checks
+from django.test import override_settings, override_system_checks
+
+
+class SessionSettingsTests(unittest.TestCase):
+
+    @override_system_checks([check_names_conflict])
```

Comment:
Take a look at the pattern used in `tests/check_framework/test_templates.py` -- that doesn't require `@override_system_checks` and is a bit cleaner I think. (made this comment last time, but it got hidden)